{{cookiecutter.project_shortname}}/setup.py

```diff
@@ -3,7 +3,7 @@
 from setuptools import setup
 
 
-with open(os.path.join('{{cookiecutter.project_shortname}}', 'package.json')) as f:
+with open(os.path.join('{{cookiecutter.project_shortname}}', 'package-info.json')) as f:
```

Comment:
```suggestion
with open('package.json') as f:
```

Good catch @LuAPi - and my apologies for the slow review. Let's go with the simpler version. It matches what we do in the rest of the core (eg [dcc](https://github.com/plotly/dash-core-components/blob/474f196d285e83cbe750bbbab998b220a0d61ee2/setup.py#L4)) and at some point - when `__init__.py` is entirely auto-generated instead of generated once and then manually edited - we're intending to get rid of the copy `package-info.json` entirely.